agent/src/main/java/com/cloud/agent/IAgentShell.java

```diff
@@ -37,14 +37,12 @@ public interface IAgentShell {
 
     String getNextHost();
 
-    String getPrivateIp();
+
```

Comment:
@MANN3005 , what is your reason to remove these from the interface?

Reply:
Refactored it using push down method and  Moved it to the implementation of that

Reply:
That is how you did it. I get that. But why do you think this is an improvement?

Reply:
It's because that has only 1 implementation and it in the file where I moved it so to remove unnecessary abstraction

Reply:
Ok, this change mandates that this be extensively tested for regressions. The rest of the PR seems trivial.